bitnami/airflow-worker/2/debian-11/rootfs/opt/bitnami/scripts/libairflowworker.sh

```diff
@@ -46,6 +46,11 @@ airflow_worker_validate() {
 #   None
 #########################
 airflow_worker_initialize() {
+    # Remove airflow-worker.pid file if exists to prevent error after WSL restarts
+	if [ -f /opt/bitnami/airflow/tmp/airflow-worker.pid ]; then
+		rm /opt/bitnami/airflow/tmp/airflow-worker.pid
+	fi
```

Comment:
It seems an issue using tabs VS spaces
```suggestion
    # Remove airflow-worker.pid file if exists to prevent error after WSL restarts
    if [ -f /opt/bitnami/airflow/tmp/airflow-worker.pid ]; then
        rm /opt/bitnami/airflow/tmp/airflow-worker.pid
    fi
```